sys/mips/include/regnum.h

```diff
@@ -133,7 +133,15 @@
 #define	MULHI	34
 #define	BADVADDR 35
 #define	CAUSE	36
+#if __has_feature(capabilities)
+#define	UNUSED_PC_SLOT	37
+/* Ensure that we don't use the PC macro for CHERI, use PCC instead */
+#pragma GCC poison PC
+#else
 #define	PC	37
+#endif
+/* Define a macro that can be used for ptrace() PT_GETREGS/PT_SETREGS */
+#define	PT_REGS_PC	37
```

Comment:
This is part of the public API (for better or for worse) and is used in asan as well.  I think you just need to leave this as PC always instead.

Reply:
Okay, I'll change the poison to be guarded by `_KERNEL`. It shouldn't matter for userspace.